content/web-frameworks/django/docker-compose-for-django-mysql.md

```diff
@@ -59,18 +59,20 @@ services:
 
   db:
     container_name: mysql-server
-    image: mysql:latest
+    image: mysql:8.0.26
```

Comment:
@dmakeroam Same as previous question on MySQL docker, should it be mysql:8 for flexibility?

Reply:
Yes